preliz/unidimensional/quartile_int.py

```diff
@@ -46,171 +46,176 @@ def quartile_int(q1=1, q2=2, q3=3, dist_names=None, figsize=None):
     * See quartile mode http://optics.eee.nottingham.ac.uk/match/uncertainty.php
     """
 
-    check_inside_notebook(need_widget=True)
+    def __init__(self, q1=1, q2=2, q3=3, dist_names=None, figsize=None):
+        self._q1 = q1
+        self._q2 = q2
+        self._q3 = q3
+        self.dist = None
```

Comment:
we need to store the distribution that we fit, this is the main reason for this refactor